src/com/twitter/intellij/pants/service/project/PythonVenvBuilder.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package com.twitter.intellij.pants.service.project;
+
+import com.intellij.openapi.diagnostic.Logger;
+import com.twitter.intellij.pants.service.project.model.PythonInterpreterInfo;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+public class PythonVenvBuilder {
+
+  private final Path executable;
+
+  private static final String executableName = "venv_builder.pex";
```

Comment:
I'm wondering if we could run here just `./pants run ...` (this would build&run the exec)

Reply:
Done